src/NewProject/new_project_dialog.cpp

```diff
@@ -49,25 +89,29 @@ QString newProjectDialog::getProject() {
 
 void newProjectDialog::Reset() {
   m_index = INDEX_LOCATION;
-  for (int i = 0; i < ui->m_stackedWidget->count(); i++) {
-    auto w = ui->m_stackedWidget->widget(i);
-    ui->m_stackedWidget->removeWidget(w);
-    delete w;
-  }
+  ui->m_tabWidget->clear();
```

Comment:
This change will probably result in the tabs of m_tabWidget [not getting deleted ](https://doc.qt.io/qt-6/qtabwidget.html#clear)on reset, is that intentional?

Reply:
That's intentional, yes - widgets are created with parents and will be deleted anyway. 
Still, it might be that after removing them from QTabWidget parent gets reset and we get a memory leak indeed. Will check that.

Reply:
![image](https://user-images.githubusercontent.com/109239890/192273697-38536882-82b7-440b-bfe2-e4dfd693c92e.png)

Confirmed - forms remain 'parented' after clearing the tab widget. They will be removed by QObject parent mechanism when the dialog gets destroyed - with current implementation at application exit.

Reply:
so technically this is no memory leak but it behaves like that. Every time we open this widget we use new memory and not free the one we don't need.